tensilelite/Tensile/Source/lib/include/Tensile/ContractionProblemPredicates.hpp

```diff
@@ -1981,7 +1981,9 @@ namespace Tensile
 
                 virtual bool operator()(ContractionProblemGemm const& problem) const override
                 {
-                    return problem.activationType() == value || value == ActivationType::All;
+                    return problem.activationType() == value
+                           || value == ActivationType::All
+                           || value == ActivationType::Hipblaslt_all;
```

Comment:
for Hipblaslt_all, need to check if the problem's activation is in hipblaslt act supported list.

Reply:
Fixed in the new patch set. Please help to review.

Reply:
The new patch set version will cause a lot of failure:
Failed
NO solution found!

Further discussion is needed to determine the correct solution.